src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public abstract class Figure {
+    private String color;
+    private double area;
```

Comment:
Consider changing the `area` field to be calculated dynamically in subclasses rather than storing it as a field, to avoid inconsistencies.